docs/quickstart/apollo_2_5_hardware_system_installation_guide_v1.md

```diff
@@ -214,6 +216,19 @@ For more information about the NovAtel GPS-703-GGG-HV, see:
 
 [https://www.novatel.com/products/gnss-antennas/high-performance-gnss-antennas/gps-703-ggg-hv/](https://www.novatel.com/products/gnss-antennas/high-performance-gnss-antennas/gps-703-ggg-hv/)
 
+### Option 2: **Dual NovAtel GNSS-502** 
```

Comment:
Do we need to re-config PP6 on vehicle side for this dual antenna option or any code changes needed? @terrylamprecht